src/style.css

```diff
@@ -1,3 +1,108 @@
-body {
+html {
+  --color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use the `*` selector as it impacts performance. It is better to set styles only for elements that require them.